### PR TITLE
Define Applicatives by `liftA2` and add default (>>)

### PR DESCRIPTION
**Pull request by b-mehta**
Added some minimal pragmas as in standard base to allow alternate definition of Applicative. Added default definition of `(>>)`.

### CODE REVIEW

File: src/Data/Functor/Linear.hs [hunk 1]
```diff
@@ -28,7 +29,7 @@ class Functor f where
 -- drops values, which we are not allowed to do in a linear container).
 --
 -- In fact, an applicative functor is precisely a functor equipped with (pure
--- and) @zipWith :: (a ->. b ->. c) -> f a ->. f b ->. f c@
+-- and) @liftA2 :: (a ->. b ->. c) -> f a ->. f b ->. f c@
```
Comment (aspiwack) [inline src/Data/Functor/Linear.hs]: Well, the reference to `zipWith` was rather on purpose. We probably want to say that `liftA2` has the same signature as `zipWith` does for lists or something.

**Comment by aspiwack** [comment]
Mmm… I see that there is no CI running. I'll have to remedy that.

Also: don't forget to delete branch after merging.

**Comment by b-mehta** [comment]
It looks like CI did run here: https://circleci.com/gh/tweag/linear-base/107, but it wasn't reported for some reason.

**Comment by b-mehta** [comment]
It turns out github status updates were off for this repo, I've turned them back on and CircleCI now reports the results of checks.

**Pull request merged**